docs/source/getting_started.rst

```diff
@@ -36,20 +36,49 @@ in various programming languages. Smart contract APIs are available for Go, Node
 Hyperledger Fabric application SDKs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Hyperledger Fabric offers a number of SDKs to support developing applications
-in various programming languages. SDKs are available for Node.js and Java:
+Hyperledger Fabric offers a number of SDKs to support developing applications in various programming languages. SDKs are available for Node.js and Java:
 
   * `Node.js SDK <https://github.com/hyperledger/fabric-sdk-node>`__ and `Node.js SDK documentation <https://hyperledger.github.io/fabric-sdk-node/>`__.
   * `Java SDK <https://github.com/hyperledger/fabric-gateway-java>`__ and `Java SDK documentation <https://hyperledger.github.io/fabric-gateway-java/>`__.
 
+  If you will be developing applications for Hyperledger Fabric leveraging the
+  Hyperledger Fabric SDK for Node.js, version 8 is supported from 8.9.4 and higher.
+  Node.js version 10 is supported from 10.15.3 and higher.
+
+    - `Node.js <https://nodejs.org/en/download/>`__ download
+
+  .. note:: Installing Node.js will also install NPM, however it is recommended
+            that you confirm the version of NPM installed. You can upgrade
+            the ``npm`` tool with the following command:
+
+  .. code:: bash
+  
+    npm install npm@5.6.0 -g
+
 In addition, there are two more application SDKs that have not yet been officially released
 (for Python and Go), but they are still available for downloading and testing:
 
   * `Python SDK <https://github.com/hyperledger/fabric-sdk-py>`__.
   * `Go SDK <https://github.com/hyperledger/fabric-sdk-go>`__.
 
-Currently, Node.js and Java support the new application programming model delivered in
-Hyperledger Fabric v1.4. Support for Go is planned to be delivered in a later release.
+Currently, Node.js and Java support the new application programming model delivered in Hyperledger Fabric v1.4. Support for Go is planned to be delivered in a later release.
+
+.. note:: The following applies to Ubuntu 16.04 users only.
+
+By default Ubuntu 16.04 comes with Python 3.5.1 installed as the ``python3`` binary.
+The Fabric Node.js SDK requires an iteration of Python 2.7 in order for ``npm install``
+operations to complete successfully.  Retrieve the 2.7 version with the following command:
+
+.. code:: bash
+
+  sudo apt-get install python
+
+Check your version(s):
+
+.. code:: bash
+
+  python --version
+
```

Comment:
I believe this Python section only applies to Node.js SDK. I'm not sure why Python is needed actually or if that is still the case. Anyways, this section should also move to Node.js SDK docs so that the Node.js experts can maintain and update it.

Reply:
For reference - Node.js uses python to build native modules (i.e. grpc)

Reply:
@mbwhite Python is not listed as a pre-req in the Node.js readme. Should it be added? https://github.com/hyperledger/fabric-sdk-node#build-and-test

Reply:
Per Matthew White's agreement, I removed this section from the page.